engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py

```diff
@@ -59,6 +59,22 @@ class VllmRbProperties(Properties):
     enable_prefix_caching: Optional[bool] = False
     disable_sliding_window: Optional[bool] = False
     limit_mm_per_prompt: Optional[Mapping[str, int]] = None
+    use_v2_block_manager: bool = False
```

Comment:
FYI: https://github.com/vllm-project/vllm/pull/8678 (not merged yet, but changes this default to True in vLLM).